src/test/scala/mesosphere/marathon/core/instance/InstanceTestHelper.scala

```diff
@@ -0,0 +1,12 @@
+package mesosphere.marathon.core.instance
+
+import mesosphere.marathon.core.task.Task
+
+import scala.collection.immutable.Seq
+
+trait InstanceTestHelper {
```

Comment:
I didn't want to pollute the non-test API with helper code needed for tests. Is there a better plan/pattern for such?